add guidelines and replace "Evidence" with "Argument" across the docs (#131)
polkadot-fellows/Evaluations: README.md

# Evaluations

This repository contains all the Arguments filed as part of the Retention and Promotion of members of the Polkadot Technical Fellowship. These Arguments have been submitted for the discussion and review of individual members' contributions and achievements, as well as for the Fellowship's on-chain bodies to signal approval or disapproval of.

More information about Membership management can be found on the [Polkadot Technical Fellowship Dashboard](https://polkadot-fellows.xyz/#/membership).

## History

> Arguments used to be called simply "Evidence" and you may sometimes see this defunct term used. It was deprecated in favour of "Arguments" since people misconstrued their responsibilities regarding this information and tended to simply submit a list of code changes, which is unacceptable.

Reviewer: ```suggestion
Arguments used to be called simply "Evidence" and you may sometimes see this defunct term used. It was deprecated in favour of "Arguments", since people misconstrued their responsibilities regarding this information and tended to simply submit a list of code changes.
```

 * ‘subxt‘ (useful tooling, but not required for Polkadot’s continued operation); and
 * ‘ink!’ (useful tooling, but not required for Polkadot’s continued operation).

Arguments are scoped to the subset of these concerns which must be held consistent across all evaluations for Retention and Promotion.

## Guidelines

Arguments may be made by members or candidates for the purpose of rank retention or promotion.

> Arguments must directly address each of the Rank Requirements as stated in the Manifesto for the rank to be retained or promoted to. If in doubt, copy and paste each requirement as stated in the Manifesto and write a short form answer, making links where necessary to prove of your activity. Ensure any proof is __specific__ to the Manifesto requirements and __substantial__. Irrelevant or insubstantial proofs may result in an early dismissal of the proposal.

Reviewer: ```suggestion
Arguments must directly address each of the Rank Requirements as stated in the Manifesto for the rank at which to be retained or at which to be promoted. If in doubt, copy and paste each requirement as stated in the Manifesto and write a short form answer, making links where necessary to demonstrate your activity. Ensure any proof is __specific__ to the Manifesto requirements and __substantial__. Irrelevant or insubstantial proofs may result in an early dismissal of the proposal.
```

## Process

> The process for submitting Arguments is open to all existing Fellowship members (i.e Rank I to IX). Anyone may provide comments on submitted Arguments.

Reviewer: ```suggestion
The process for submitting Arguments is open to all existing Fellowship members (i.e. Rank I to IX). Anyone may provide comments on submitted Arguments.
```
